packages/lit-dev-content/site/docs/templates/expressions.md

```diff
@@ -137,11 +137,11 @@ html`<main>${bodyText}</main>`
 
 Expressions in the child position can take many kinds of values:
 
-* Primitive values
-* Sentinel values
-* `TemplateResult` objects created with the `html` function.
-* DOM nodes
-* Arrays or iterables of any of the supported types
+* Primitive values likes strings, numbers, and booleans.
+* `TemplateResult` objects created with the [`html`](/docs/api/templates/#html) or [`svg`](/docs/api/templates/#svg) functions.
```

Comment:
```suggestion
* `TemplateResult` objects created with the [`html`](/docs/api/templates/#html) or [`svg`](/docs/api/templates/#svg) functions, when inside an `svg` element.
```

Seems most important to call this out here. I'm meh on putting it in the other spot as well.

Reply:
Like the idea, but this way it's not clear that "when inside an `svg` element" applies only to the `svg` function. Maybe:

```suggestion
* `TemplateResult` objects created with the [`html`](/docs/api/templates/#html) function (or the [`svg`](/docs/api/templates/#svg) function, if the expression is inside an `<svg>` element).
```